packages/twenty-front/src/pages/settings/developers/webhooks/components/SettingsDevelopersWebhookDetail.tsx

```diff
@@ -282,8 +282,16 @@ export const SettingsDevelopersWebhooksDetail = () => {
         </Section>
         {isAnalyticsEnabled && isAnalyticsV2Enabled && (
           <>
-            <SettingsDevelopersWebhookUsageGraphEffect webhookId={webhookId} />
-            <SettingsDevelopersWebhookUsageGraph webhookId={webhookId} />
+            <AnalyticsGraphEffect
+              recordId={webhookId}
+              recordType="webhook"
+              endpointName="getWebhookAnalytics"
+            />
+            <AnalyticsActivityGraph
+              recordId={webhookId}
+              recordType="webhook"
```

Comment:
We can have a map (constant) that stores the record type for each endpoint, instead of passing it downstream here. GetWebhookAnalytics will always be with the webhook recordType.